model_prices_and_context_window.json

```diff
@@ -7477,6 +7477,30 @@
         "litellm_provider": "bedrock",
         "mode": "embedding"
     },
+    "us.deepseek.r1-v1:0": {
+        "max_tokens": 4096, 
+        "max_input_tokens": 128000,
+        "max_output_tokens": 4096,
+        "input_cost_per_token": 0.00000135,
+        "output_cost_per_token": 0.0000054,
+        "litellm_provider": "bedrock_converse",
+        "mode": "chat",
+        "supports_function_calling": false, 
+        "supports_tool_choice": false
+
+    },
+    "eu.deepseek.r1-v1:0": {
```

Comment:
sorry, can you remove this. I don't see Bedrock with eu pricing as of now https://aws.amazon.com/bedrock/pricing/

Reply:
bump @kearnsw